pages/blog/changes-coming-docs.md

```diff
@@ -0,0 +1,149 @@
+---
+title: "Change is coming to our AsyncAPI Developer Documentation"
+Subtitle: "AsyncAPI Dev Docs are getting a makeover!"
+date: 2021
+type: Engineering
+tags:
+  - Documentation
+  - Announcement
+cover: /img/posts/changes-coming-docs/diataxis.jpg
+authors:
+  - name: Alejandra Quetzalli
+    photo: /img/avatars/canela-ale.jpg
+    link: https://www.linkedin.com/in/alejandra-quetzalli/
+    byline: AsyncAPI Dev Docs are getting a makeover!
+excerpt: Changes are coming to the Docs!
+---
+
+## 🦄 Hola, soy Alejandra Quetzalli
+
+¡Hola! 😄
+
+AsyncAPI community, it's an absolute pleasure to meet you. My name is Alejandra. I was born and raised in México 🇲🇽, so my first language is Spanish.
+
+I’m a Senior Technical Writer 👩🏻‍💻 recently hired by [Postman](https://www.postman.com/alejandra-quetzalli) to focus solely on the Open-Source (OSS) [AsyncAPI initiative](https://www.asyncapi.com/) as primary owner for our AsyncAPI Dev Docs 📄 . 
+
+I also recently [joined our initiative's Technical Steering Committee (TSC)](https://www.asyncapi.com/community/tsc)! The TSC is responsible for the oversight of the AsyncAPI Initiative, helping make decisions on a higher level, or when maintainers cannot find a consensus.
+
+By the second day on the team, I was already working on community Pull Requests (PRs) and getting to know more of our community members. This direct engagement with community feels amazing, because I really **_do_** want to help as much as I can. I'm excited to listen what our OSS community thinks needs to be documented first.
+
+Before I tell you more details about upcoming changes to the Docs and how you can contribute to them, I also want to share more granular details about my work here within our OSS community.
+
+***
+
+### 👩🏻‍💻 What I do for AsyncAPI Docs
+- **Own the docs for the AsyncAPI feature set** — documenting this specific area of our Open Technologies function and driving all docs efforts around it.
+- **Plan documentation in conjunction with OSS community feedback** — working with open-source communities to learn about a feature from specifications and user research.
+- **Collaborate closely with Developer Relations to ensure docs, educational, and learning materials align with community needs:** this includes assisting with documentation, tutorials, and all education efforts within AsyncAPI.
+- **Conduct editorial reviews on my teammates’ and community doc drafts**— providing constructive and kind feedback that helps colleagues to grow.
```

Comment:
there should not be a differentiation between community and teammates 😉

Reply:
Right, I keep feeling confused on this nuance.. fixing in next commit!